### PR TITLE
adding labels for the Refresh Administrative|Household Account Names …

### PR DESCRIPTION
**Pull request by lifewithryan**
…buttons in System tab of EDA settings so they can be translated rather than hard-coding the labels

# Critical Changes

# Changes
- We've added two custom labels to be used by our Refresh All Administrative Names, and Refresh All Household Names buttons on the System Tab of EDA Settings. This was done to ensure that those buttons can be translated.
# Issues Closed

# New Metadata
- Custom labels: stgBtnRefreshAdminNames, stgBtnRefreshHouseholdNames
# Deleted Metadata

# Testing Notes
- Open EDA settings and ensure the buttons in question still read Refresh All Admin Names, and Refresh All Household Names (the translations themselves will be provided in a later release).

### CODE REVIEW

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -847,6 +847,22 @@
         <shortDescription>Button label for multiple EDA Settings tabs</shortDescription>
         <value>Edit</value>
     </labels>
+    <labels>
+        <fullName>stgBtnRefreshAdminNames</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label for refreshing Administrative Account Names</shortDescription>
```
Comment (xinnanlisf) [inline src/labels/CustomLabels.labels:855]: Not sure if we want get Doc involved on these description. Let me know if you already have.

File: src/aura/STG_CMP_System/STG_CMP_System.cmp [hunk 2]
```diff
@@ -48,8 +48,8 @@
                         </div>
 
                          <div class="slds-col slds-align-middle slds-gutters">
-                        	<h2 class="slds-text-heading_small">{!$Label.c.stgSuccessAdminAccountNameRefresh}</h2>
-                       	</div>
+                            <h2 class="slds-text-heading_small">{!$Label.c.stgSuccessAdminAccountNameRefresh}</h2>
```
Comment (xinnanlisf) [inline src/aura/STG_CMP_System/STG_CMP_System.cmp:51]: Spacing looks a little odd here.

File: src/aura/STG_CMP_System/STG_CMP_System.cmp [hunk 3]
```diff
@@ -252,19 +252,19 @@
...
                     {!$Label.c.stgRefreshAdminAcctNameTitle}
                     <div class="slds-col slds-size--1-of-1">
-                    	<ui:outputText value="{!$Label.c.stgRefreshAdminAcctNameDesc}" class="slds-text-body--small" />
+                        <ui:outputText value="{!$Label.c.stgRefreshAdminAcctNameDesc}" class="slds-text-body--small" />
                     </div>
                 </div>
 
-              	<div class="slds-col slds-size--1-of-2">
-              		<lightning:button
+                  <div class="slds-col slds-size--1-of-2">
```
Comment (xinnanlisf) [inline src/aura/STG_CMP_System/STG_CMP_System.cmp:259]: Spacing not aligned here either.

File: src/aura/STG_CMP_System/STG_CMP_System.cmp [hunk 4]
```diff
@@ -252,19 +252,19 @@
...
+                      <lightning:button
                                 disabled="{!!v.isView}"
-                        	    variant="brand"
-                    		    label="Refresh All Administrative Names"
-                        	    title="refreshButton"
-                        	    onclick="{! c.handleRefreshAdminAccount}"/>
+                                variant="brand"
+                                label="{!$Label.c.stgBtnRefreshAdminNames}"
+                                title="refreshButton"
+                                onclick="{! c.handleRefreshAdminAccount}"/>
```
Comment (xinnanlisf) [inline src/aura/STG_CMP_System/STG_CMP_System.cmp:265]: Not quite related to this item but maybe we can use the chance to get rid of the extra spaces here. There is another place too.

File: src/labels/CustomLabels.labels (hunk 1 above)
Comment (lifewithryan) [reply src/labels/CustomLabels.labels:855]: I tried to keep in sync with what we'd been doing historically. I think the risk is low given that this only appears in setup with custom labels and not really 'in the customer's face'
I have not involved doc as I felt that would be overkill.

File: src/aura/STG_CMP_System/STG_CMP_System.cmp (hunk 4 above)
Comment (lifewithryan) [reply src/aura/STG_CMP_System/STG_CMP_System.cmp:265]: I'll take a look, will have to wait until tomorrow though.

File: src/aura/STG_CMP_System/STG_CMP_System.cmp (hunk 3 above)
Comment (lifewithryan) [reply src/aura/STG_CMP_System/STG_CMP_System.cmp:259]: I'll take a look, will have to wait until tomorrow though.

File: src/aura/STG_CMP_System/STG_CMP_System.cmp (hunk 2 above)
Comment (lifewithryan) [reply src/aura/STG_CMP_System/STG_CMP_System.cmp:51]: I'll take a look, will have to wait until tomorrow though.

**Comment by lifewithryan** [comment]
> @lifewithryan tested and worked well. I did make a few comments on some minor spacing. Also, I thought we wanted to include translation files for custom labels?

Do we need to add the translation placeholders into the CustomLabel translation files? I was thinking that the translation team's software would add that. I could be wrong.

**Comment by lifewithryan** [comment]
@xinnanlisf @MatthewBlanski -- I tried to pull the translation placeholder stuff down and everything just lit up. There were all sorts of quickAction areas in the *.translation files as well which didn't seem right. I pinged Matt and I think we can skip that part for this push. Maybe we can fix this and add the placeholders separately if it is a big deal. Otherwise, maybe it's just me doing something wrong and we want someone else to look at the placeholders.

**Comment by lifewithryan** [comment]
> @xinnanlisf @MatthewBlanski -- I tried to pull the translation placeholder stuff down and everything just lit up. There were all sorts of quickAction areas in the *.translation files as well which didn't seem right. I pinged Matt and I think we can skip that part for this push. Maybe we can fix this and add the placeholders separately if it is a big deal. Otherwise, maybe it's just me doing something wrong and we want someone else to look at the placeholders.

OKay looks like I can perhaps try this manually. stand by

File: src/translations/ca.translation [hunk 5]
```diff
@@ -428,6 +428,14 @@
         <label>Edita</label>
         <name>stgBtnEdit</name>
     </customLabels>
+    <customLabels>
+        <label><!-- Refresh All Administrative Names --></label>
+        <name>stgBtnRefreshAdminNames</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- Refresg All Household Names --></label>
```
Comment (MatthewBlanski) [inline src/translations/ca.translation:436]:
```suggestion
        <label><!-- Refresh All Household Names --></label>
```

File: src/translations/en_GB.translation [hunk 6]
```diff
@@ -428,6 +428,14 @@
         <label>Edit</label>
         <name>stgBtnEdit</name>
     </customLabels>
+    <customLabels>
+        <label><!-- Refresh All Administrative Names --></label>
+        <name>stgBtnRefreshAdminNames</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- Refresg All Household Names --></label>
```
Comment (MatthewBlanski) [inline src/translations/en_GB.translation:436]:
```suggestion
        <label><!-- Refresh All Household Names --></label>
```

File: src/translations/es.translation [hunk 7]
```diff
@@ -428,6 +428,14 @@
         <label>Modificar</label>
         <name>stgBtnEdit</name>
     </customLabels>
+    <customLabels>
+        <label><!-- Refresh All Administrative Names --></label>
+        <name>stgBtnRefreshAdminNames</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- Refresg All Household Names --></label>
```
Comment (MatthewBlanski) [inline src/translations/es.translation:436]:
```suggestion
        <label><!-- Refresh All Household Names --></label>
```

File: src/translations/es_MX.translation [hunk 8]
```diff
@@ -428,6 +428,14 @@
         <label>Modificar</label>
         <name>stgBtnEdit</name>
     </customLabels>
+    <customLabels>
+        <label><!-- Refresh All Administrative Names --></label>
+        <name>stgBtnRefreshAdminNames</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- Refresg All Household Names --></label>
```
Comment (MatthewBlanski) [inline src/translations/es_MX.translation:436]:
```suggestion
        <label><!-- Refresh All Household Names --></label>
```

File: src/translations/fr.translation [hunk 9]
```diff
@@ -428,6 +428,14 @@
         <label>Modifier</label>
         <name>stgBtnEdit</name>
     </customLabels>
+    <customLabels>
+        <label><!-- Refresh All Administrative Names --></label>
+        <name>stgBtnRefreshAdminNames</name>
+    </customLabels>
+    <customLabels>
+        <label><!-- Refresg All Household Names --></label>
```
Comment (MatthewBlanski) [inline src/translations/fr.translation:436]:
```suggestion
        <label><!-- Refresh All Household Names --></label>
```

**Comment by dgamboni** [comment]
Tested/Passes QE.

**Pull request merged**